pkgs/tools/misc/usbimager/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, stdenv, fetchFromGitLab, pkg-config
+, withLibui ? true, gtk3
+, withUdisks ? false, udisks, glib
```

Comment:
Is any of this conditional on Linux?

Reply:
(If I understand the question correctly) Yes, udisks is linux-only, do I have to take care of that in this package?

Reply:
yes

found this example. not sure if that's the best practice, but looks like it could work https://github.com/NixOS/nixpkgs/commit/8ca70e358d8836c66153e85db2aee4a0fd44fd21